src/features/instance/operations/queries/getSearchByHash.ts

```diff
@@ -1,37 +1,15 @@
 import { instanceClient } from '@/config/instanceClient';
 import { queryOptions } from '@tanstack/react-query';
 
-// type SearchConditions = {
-// 	search_attribute: string;
-// 	search_type: string;
-// 	search_value: string;
-// };
-
-// type SearchByValueRequest = {
-// 	conditions?: [SearchConditions];
-// 	schema: string;
-// 	table: string;
-// 	search_attribute: string;
-// 	search_value: string;
-// 	// get_attributes: string[];
-// 	limit: number;
-// 	offset: number;
-// };
-function getSearchByHashOptions({
-	instanceId,
-	schemaName,
-	tableName,
-	hashAttribute,
-}: // ...options
-{
-	instanceId: string;
-	schemaName: string;
-	tableName: string;
-	hashAttribute: string[];
-	// options?: SearchByValueRequest;
-}) {
+function getSearchByHashOptions(
+	isEditModalOpen: boolean,
+	instanceId: string,
+	schemaName: string,
+	tableName: string,
+	hashAttribute?: unknown,
+) {
 	return queryOptions({
-		queryKey: [instanceId, 'search_by_hash'] as const,
+		queryKey: ['search_by_hash', instanceId, schemaName, tableName, hashAttribute] as const,
```

Comment:
Minor nit, but can you change `search_by_hash` to `search_by_id`? We changed the name of this operation a couple years ago (hash was an inaccurate name). It is also more correct to use `primaryKey` than `hashAttribute`, although that's just an internal variable here.

Reply:
@kriszyp Done!